manifests/r/RealVNC/VNCServer/6.9.0.46599/RealVNC.VNCServer.installer.yaml

```diff
@@ -0,0 +1,21 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.7-2-2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: RealVNC.VNCServer
+PackageVersion: 6.9.0.46599
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /qn REBOOT=ReallySuppress
+  SilentWithProgress: /qn REBOOT=ReallySuppress
+UpgradeBehavior: install
+Installers:
+- Architecture: neutral
+  InstallerUrl: https://downloads.realvnc.com/download/file/vnc.files/VNC-Server-6.9.0-Windows.exe
+  InstallerSha256: B41C2EAF01C71F56332708DF4E395F1257DE16421C8586669AFD501E0754D4A7
```

Comment:
The website (https://www.realvnc.com/en/connect/download/vnc/windows/) says x86/x64 so i think it's both

Reply:
Ahh okay good, the upside with doing 2 different Architecture is that we can spot when they drop support for x86, also i think neutral also includes arm/arm64 (which is why we've moved away from neutral(or mainly because it ONLY tests on x64))